kwok/user-signups.sh

```diff
@@ -3,7 +3,7 @@
 namespace="toolchain-host-operator"
 kwokctl --name "$CLUSTER_NAME" kubectl create namespace "$namespace"
 
-for i in $(seq 1 10)
+for i in $(seq 1 9)
```

Comment:
What's the reason for using `9` specifically? might be better to include it as a named variable.

Reply:
I am also setting up sso-id with this.

Reply:
all the more reason to include it as a named variable.